.github/workflows/ci.yaml

```diff
@@ -68,17 +68,17 @@ jobs:
           echo
 
           echo "Obtaining the latest metadata validation tool:"
```

Comment:
should this be "metadata mapping creation tool" instead of "metadata validation tool"?